jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/kafka/topics/JsonTopics.java

```diff
@@ -4,33 +4,37 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+/**
+ * Configuration class for JSON-formatted Kafka topics. Prefix: ode.kafka.topics.json
```

Comment:
suggestion(non-blocking): since the prefix is already available on line 11 below, we should remove the duplicated knowledge to follow the DRY principle and prevent accidental drift
```suggestion
 * Configuration class for JSON-formatted Kafka topics.
```

Reply:
addressed